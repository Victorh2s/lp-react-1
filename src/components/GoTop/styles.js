import styled, { css } from 'styled-components';

export const Container = styled.a`
  ${({ theme }) => css`
    position: fixed;
    background: none;
    color: ${theme.colors.colorbuttons};
    border: 0.1rem solid ${theme.colors.colorbuttons};
    border-radius: 5rem;
    display: flex;
    align-items: center;
    justify-content: center;
    width: 4rem;
    height: 4rem;
    bottom: 2rem;
    right: 2rem;
    z-index: 6;
  `}
`;
