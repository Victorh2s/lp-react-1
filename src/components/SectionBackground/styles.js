import styled, { css } from 'styled-components';

const backgroundActive = (theme) => css`
  background: ${theme.colors.primaryColor};
  color: ${theme.colors.white};
`;

export const Container = styled.div`
${({ theme, background }) => css`
transition: all 300ms ease-in-out;
background: ${theme.colors.white};
color: ${theme.colors.primaryColor};
${background && backgroundActive(theme)}
min-height: 80vh;
display: flex;
align-items: center;
`}
`;
