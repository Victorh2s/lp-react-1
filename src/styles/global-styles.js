import { createGlobalStyle, css } from 'styled-components';

export const GlobalStyles = createGlobalStyle`
  @import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,400;0,600;0,800;1,700&display=swap');
  * {
    margin:0;
    padding:0;
  }
  html {
    font-size: 62.5%;
  }
  body {
    font-size:2.4rem;
    font-family: ${({theme})=> theme.font,family.default};
  }
  h1 {
    font-size: 2.4rem;
    font-family:  ${({theme})=> theme.font,family.secondary};
  }
`;
