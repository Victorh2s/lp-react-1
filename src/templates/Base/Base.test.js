import { screen } from '@testing-library/react';
import { Base } from '.';
import { renderTheme } from '../../styles/render-theme';

describe('<Base />', () => {
  it('should render', () => {
    renderTheme(<Base>Children</Base>);
    const Base = screen.getByRole('heading');
    expect(Base).toBeInTheDocument();
  });
});
