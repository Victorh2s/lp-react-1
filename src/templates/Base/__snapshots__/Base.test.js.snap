// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Base /> should render 1`] = `
<div>
  <button
    aria-label="Open/Close menu"
    class="sc-jqUVSM eWDHFI"
  >
    <svg
      aria-hidden="true"
      aria-label="Open menu"
      class="StyledIconBase-ea9ulj-0 lbJwfL"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
      <path
        d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
      />
    </svg>
  </button>
  <div
    class="sc-ftvSup dnLJaK"
  >
    <div
      class="sc-jSMfEi dRbZhr"
    >
      <div
        class="sc-papXJ eIiUmt"
      >
        <h2
          class="sc-bczRLJ kukvte"
        >
          <a
            class="sc-kDDrLX egbyVN"
            href="#"
          >
            Logo
          </a>
        </h2>
        <nav
          class="sc-iqcoie gVkxmo"
        >
          <a
            class="sc-crXcEl cxixyS"
            href="#target1"
            target="_self"
          >
            link 1
          </a>
          <a
            class="sc-crXcEl cxixyS"
            href="#target2"
            target="_self"
          >
            link 2
          </a>
          <a
            class="sc-crXcEl cxixyS"
            href="#target3"
            target="_self"
          >
            link 3
          </a>
          <a
            class="sc-crXcEl cxixyS"
            href="#target4"
            target="_self"
          >
            link 4
          </a>
          <a
            class="sc-crXcEl cxixyS"
            href="#target5"
            target="_self"
          >
            link 5
          </a>
          <a
            class="sc-crXcEl cxixyS"
            href="#target6"
            target="_self"
          >
            link 6
          </a>
          <a
            class="sc-crXcEl cxixyS"
            href="#target7"
            target="_self"
          >
            link 7
          </a>
          <a
            class="sc-crXcEl cxixyS"
            href="#target8"
            target="_self"
          >
            link 8
          </a>
          <a
            class="sc-crXcEl cxixyS"
            href="#target9"
            target="_self"
          >
            link 9
          </a>
          <a
            class="sc-crXcEl cxixyS"
            href="#target10"
            target="_self"
          >
            link 10
          </a>
        </nav>
      </div>
    </div>
  </div>
  <div
    class="sc-iBkjds lobcSh"
  >
    <div
      class="sc-gKXOVf dOsZSp"
      id=""
    >
      <div
        class="sc-jSMfEi dRbZhr"
      >
        <div
          class="sc-dkzDqf heXQUQ"
        >
          <h2
            class="sc-bczRLJ kSKtEH"
          >
            My grid
          </h2>
          <div
            class="sc-gsnTZi fSQKXd"
          >
            Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
          </div>
          <div
            class="sc-hKMtZM lkscSh"
          >
            <div
              class="sc-eCYdqJ lluHNs"
            >
              <h3
                class="sc-bczRLJ llKdEO"
              >
                Teste 1
              </h3>
              <div
                class="sc-gsnTZi fSQKXd"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="sc-eCYdqJ lluHNs"
            >
              <h3
                class="sc-bczRLJ llKdEO"
              >
                Teste 2
              </h3>
              <div
                class="sc-gsnTZi fSQKXd"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="sc-eCYdqJ lluHNs"
            >
              <h3
                class="sc-bczRLJ llKdEO"
              >
                Teste 3
              </h3>
              <div
                class="sc-gsnTZi fSQKXd"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="sc-gKXOVf hMKDKT"
      id=""
    >
      <div
        class="sc-jSMfEi dRbZhr"
      >
        <div
          class="sc-dkzDqf heXQUQ"
        >
          <h2
            class="sc-bczRLJ itwqPE"
          >
            My grid
          </h2>
          <div
            class="sc-gsnTZi fSQKXd"
          >
            Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
          </div>
          <div
            class="sc-hKMtZM lkscSh"
          >
            <div
              class="sc-eCYdqJ lluHNs"
            >
              <h3
                class="sc-bczRLJ fnHBwd"
              >
                Teste 1
              </h3>
              <div
                class="sc-gsnTZi fSQKXd"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="sc-eCYdqJ lluHNs"
            >
              <h3
                class="sc-bczRLJ fnHBwd"
              >
                Teste 2
              </h3>
              <div
                class="sc-gsnTZi fSQKXd"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="sc-eCYdqJ lluHNs"
            >
              <h3
                class="sc-bczRLJ fnHBwd"
              >
                Teste 3
              </h3>
              <div
                class="sc-gsnTZi fSQKXd"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="sc-gKXOVf dOsZSp"
      id=""
    >
      <div
        class="sc-jSMfEi dRbZhr"
      >
        <div
          class="sc-dkzDqf heXQUQ"
        >
          <h2
            class="sc-bczRLJ kSKtEH"
          >
            My grid
          </h2>
          <div
            class="sc-gsnTZi fSQKXd"
          >
            Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
          </div>
          <div
            class="sc-hKMtZM lkscSh"
          >
            <div
              class="sc-eCYdqJ lluHNs"
            >
              <h3
                class="sc-bczRLJ llKdEO"
              >
                Teste 1
              </h3>
              <div
                class="sc-gsnTZi fSQKXd"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="sc-eCYdqJ lluHNs"
            >
              <h3
                class="sc-bczRLJ llKdEO"
              >
                Teste 2
              </h3>
              <div
                class="sc-gsnTZi fSQKXd"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="sc-eCYdqJ lluHNs"
            >
              <h3
                class="sc-bczRLJ llKdEO"
              >
                Teste 3
              </h3>
              <div
                class="sc-gsnTZi fSQKXd"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="sc-gKXOVf hMKDKT"
      id=""
    >
      <div
        class="sc-jSMfEi dRbZhr"
      >
        <div
          class="sc-dkzDqf heXQUQ"
        >
          <h2
            class="sc-bczRLJ itwqPE"
          >
            My grid
          </h2>
          <div
            class="sc-gsnTZi fSQKXd"
          >
            Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
          </div>
          <div
            class="sc-hKMtZM lkscSh"
          >
            <div
              class="sc-eCYdqJ lluHNs"
            >
              <h3
                class="sc-bczRLJ fnHBwd"
              >
                Teste 1
              </h3>
              <div
                class="sc-gsnTZi fSQKXd"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="sc-eCYdqJ lluHNs"
            >
              <h3
                class="sc-bczRLJ fnHBwd"
              >
                Teste 2
              </h3>
              <div
                class="sc-gsnTZi fSQKXd"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="sc-eCYdqJ lluHNs"
            >
              <h3
                class="sc-bczRLJ fnHBwd"
              >
                Teste 3
              </h3>
              <div
                class="sc-gsnTZi fSQKXd"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="sc-gKXOVf dOsZSp"
      id=""
    >
      <div
        class="sc-jSMfEi dRbZhr"
      >
        <div
          class="sc-dkzDqf heXQUQ"
        >
          <h2
            class="sc-bczRLJ kSKtEH"
          >
            My grid
          </h2>
          <div
            class="sc-gsnTZi fSQKXd"
          >
            Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
          </div>
          <div
            class="sc-hKMtZM lkscSh"
          >
            <div
              class="sc-eCYdqJ lluHNs"
            >
              <h3
                class="sc-bczRLJ llKdEO"
              >
                Teste 1
              </h3>
              <div
                class="sc-gsnTZi fSQKXd"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="sc-eCYdqJ lluHNs"
            >
              <h3
                class="sc-bczRLJ llKdEO"
              >
                Teste 2
              </h3>
              <div
                class="sc-gsnTZi fSQKXd"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="sc-eCYdqJ lluHNs"
            >
              <h3
                class="sc-bczRLJ llKdEO"
              >
                Teste 3
              </h3>
              <div
                class="sc-gsnTZi fSQKXd"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="sc-gKXOVf hMKDKT"
      id=""
    >
      <div
        class="sc-jSMfEi dRbZhr"
      >
        <div
          class="sc-dkzDqf heXQUQ"
        >
          <h2
            class="sc-bczRLJ itwqPE"
          >
            My grid
          </h2>
          <div
            class="sc-gsnTZi fSQKXd"
          >
            Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
          </div>
          <div
            class="sc-hKMtZM lkscSh"
          >
            <div
              class="sc-eCYdqJ lluHNs"
            >
              <h3
                class="sc-bczRLJ fnHBwd"
              >
                Teste 1
              </h3>
              <div
                class="sc-gsnTZi fSQKXd"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="sc-eCYdqJ lluHNs"
            >
              <h3
                class="sc-bczRLJ fnHBwd"
              >
                Teste 2
              </h3>
              <div
                class="sc-gsnTZi fSQKXd"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
            <div
              class="sc-eCYdqJ lluHNs"
            >
              <h3
                class="sc-bczRLJ fnHBwd"
              >
                Teste 3
              </h3>
              <div
                class="sc-gsnTZi fSQKXd"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Debitis cum delectus molestias. Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde, reprehenderit minus ad dolores provident maiores.
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <footer
      class="sc-evZas jyzJup"
    >
      <div
        class="sc-jSMfEi dRbZhr"
      >
        <div
          class="sc-gsnTZi fSQKXd"
        >
          <p>
            Teste de footer
          </p>
        </div>
      </div>
    </footer>
  </div>
  <a
    aria-label="Go to top"
    class="sc-breuTD gQRxED"
    href="#"
    title="Go to top"
  >
    <svg
      aria-hidden="true"
      class="StyledIconBase-ea9ulj-0 lbJwfL"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
      <path
        d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6 1.41 1.41z"
      />
    </svg>
  </a>
</div>
`;
